packages/core/src/api/webhook.ts

```diff
@@ -180,12 +206,20 @@ export class WebhooksAPI {
 	 * @param id - The id of the webhook
 	 * @param token - The token of the webhook
 	 * @param messageId - The id of the message to fetch
+	 * @param query - The query options to use when fetching the message
 	 * @param options - The options to use when fetching the message
 	 */
-	public async getMessage(id: Snowflake, token: string, messageId: Snowflake, options: { thread_id?: string } = {}) {
+	public async getMessage(
+		id: Snowflake,
+		token: string,
+		messageId: Snowflake,
+		query: { thread_id?: string } = {},
```

Comment:
Isn't there a -types interface for this?

Reply:
Doesn't seem to be one.

Reply:
Lets make one then!